wsgi.py

```diff
@@ -1,9 +1,12 @@
 from dotenv import load_dotenv
 import pathlib
-load_dotenv(dotenv_path=pathlib.Path(__file__).parent/'.env.production', verbose=True)
+
+load_dotenv(dotenv_path=pathlib.Path(__file__).parent/'.env', verbose=True)
```

Comment:
should be reverted. `.env.production` is a separate file I use at the deployment time.

Reply:
Done